Dockerfile.spdk

```diff
@@ -0,0 +1,107 @@
+# syntax = docker/dockerfile:1.4
+ARG OS_RELEASE=9
+FROM registry.access.redhat.com/ubi$OS_RELEASE/ubi:latest as prebase
```

Comment:
As mentioned, I can't fully replace this, because the `pkgdep.sh` script is external to this repo (`spdk` submodule) and relies on dnf (I should replace `dnf` with `microdnf` or alias it). That said, this is only used for the build stage, so my hope is that for the final stage we'll be able to use a lighter-weight base layer.